CHANGELOG.md

```diff
@@ -40,6 +40,7 @@
     - `-experimental.tsdb.stripe-size` changed to `-experimental.blocks-storage.tsdb.stripe-size`
     - `-experimental.tsdb.wal-compression-enabled` changed to `-experimental.blocks-storage.tsdb.wal-compression-enabled`
     - `-experimental.tsdb.flush-blocks-on-shutdown` changed to `-experimental.blocks-storage.tsdb.flush-blocks-on-shutdown`
+* [CHANGE] Experimental Delete Series: `/api/v1/admin/tsdb/delete_series` and `/api/v1/admin/tsdb/cancel_delete_request` purger APIs to return status code `204` instead of `200` for success. #2946
```

Comment:
I've recently found it more difficult to use endpoint that returned 204 from scripts (curl/wget both don't like 204 very much). I understand it's "nicer" from HTTP point of view, but 200 is more pragmatic.

Reply:
Actually this is to make API compatible with Prometheuses API which returns `204`. :shrug:

Reply:
Makes sense.